src/main/java/auto_scheduler/ScheduleShower.java

```diff
@@ -1,13 +1,45 @@
 package org.txstate.auto_scheduler;
 
-public class ScheduleShower implements ScheduleAction {
-	ResourcePathProvider pathProvider;
-
-	public ScheduleShower(ResourcePathProvider pathProvider) {
-		this.pathProvider = pathProvider;
-	}
-	public ScheduleSet execute(OperationContext context)
-	{
-		return null;
-	}
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+
+
+public class ScheduleShower extends ScheduleSet implements ScheduleAction {
```

Comment:
@brominger i will do a quick commit on top of this to communicate my point across 😄 .
hopefully in a couple of horus.